similar/editor/medrobot.cpp

```diff
@@ -767,6 +767,10 @@ void object_close_window()
 //-------------------------------------------------------------------------
 int do_object_dialog()
 {
+	if (Cur_object_index == object_none)
+		Cur_object_index = object_first;
+	Assert(Cur_object_index != object_guidebot_cannot_reach);	// not supposed to be this one
```

Comment:
This assert is harmless, but unnecessary.  The call to `vobjptr` will throw if the assert would have failed.